app/src/app/add-user/add-user.component.ts

```diff
@@ -43,6 +43,10 @@ export class AddUserComponent implements OnInit {
       previousExperience: this.previousExperience.nativeElement.value,
       totalExperience: this.totalExperience.nativeElement.value
     };
+    if(this.email.nativeElement.value == "" || this.firstName.nativeElement.value == "" || this.lastName.nativeElement.value == ""||this.location.nativeElement.value == ""||this.designation.nativeElement.value == ""||this.cgiCode.nativeElement.value == ""||this.previousExperience.nativeElement.value == ""||this.totalExperience.nativeElement.value == ""||this.skills.nativeElement.value == "" && this.previousExperience.nativeElement.value > this.totalExperience.nativeElement.value){
```

Comment:
Have you tested this code ? I think this.designation.nativeElement.value will not work here.

Reply:
Yes I have changed it now!!

Reply:
You can merge it!